test-cases/gemini/gemini-1tb-10h.yaml

```diff
@@ -18,7 +18,7 @@ gemini_cmd: |
   --mode mixed
```

Comment:
If the tool is improved, then why leaving the same low throughput? or why throughput isn't improved?
The tests show results of < 2k (or 3k), why not setting a higher rate, like, for comparison, c-s 3h longevity which gets 30k?

Reply:
`Changed partition-key-buffer-reuse-size from 128 to 256 -> increases the throughput` sounds related, for example. The idea is we want to test a higher throughput. why staying with a poor throughput?